src/coffee/stockimport.coffee

```diff
@@ -371,8 +371,20 @@ class StockImport
             .fetch()
             .then (result) =>
               @_updateInventory(entry, result.body, tryCount + 1)
+            .catch (err) =>
+              if (err.statusCode == 404)
+                debug "It seem that stock update has conflicted with parallel stock deletion process."
```

Comment:
`It seems stock update has conflicted with parallel stock deletion process and can no longer be update. A new stock will be created instead`

Reply:
https://github.com/sphereio/sphere-stock-import/pull/69/commits/5b26d8671f511b68557c8f7b40cf35367ba5052e